pegasus/sites.v3/hourofcode.com/public/highlights.haml

```diff
@@ -0,0 +1,36 @@
+---
+title: Highlights
+layout: wide
+---
+
+%link{href: "/css/highlights.css", rel: "stylesheet"}
+
+%h1 Hour of Code Highlights
+%h3 Some of our favorite moments of the Hour of Code campaign.
```

Comment:
We can do later, but title and headers should be in hoc.com/i18n/en.yml and call with hoc_s(:key)